about.html

```diff
@@ -0,0 +1,377 @@
+				</div>
+			</nav>
+			<!-- navigation menu  -->
+			<nav
+				class="navbar relative bg-transparent md:bg-white md:shadow-sm"
+			>
+				<div
+					class="navbar-inner mx-auto flex items-center justify-between gap-4 py-3 md:w-10/12"
+				>
+					<button
+						id="hamburger-button"
+						class="hamburger flex-start bg h-10 cursor-pointer md:hidden"
+					>
+						<i
+							class="fa-sharp fa-solid fa-bars"
+							style="font-size: 34px"
+						></i>
+					</button>
+
+					<!-- logo  -->
+					<div>
+						<a
+							href="index.html"
+							class="logo items-center gap-2 leading-6 transition-all md:flex"
+						>
+							<img
+								src="./public/images/logo-arabic2.jpg"
+								style="width: 144px"
+								alt="logo"
+							/>
+						</a>
+					</div>
+
+					<div class="relative flex items-center justify-center">
+						<ul
+							class="hidden items-center gap-9 gap-y-4 text-xl md:flex md:flex-wrap"
+							aria-label="main"
+						>
+							<li
+								class="text-coral-red hover:text-coral-red hover:opacity-90"
+							>
+								<a href="./about.html" class=""> About </a>
+							</li>
+							<li class="hover:text-coral-red hover:opacity-90">
+								<a href="#program" class=""> Program </a>
+							</li>
+
+							<li class="hover:text-coral-red hover:opacity-90">
+								<a href="#join" class=""> Join </a>
+							</li>
+
+							<li class="hover:text-coral-red hover:opacity-90">
+								<a href="#partner" class=""> Sponsor </a>
+							</li>
+							<li class="hover:text-coral-red hover:opacity-90">
+								<a href="#" class=""> News </a>
+							</li>
+							<li
+								class="border-2 border-coral-red py-2 px-4 transition-all hover:bg-coral-red hover:text-white hover:opacity-90"
+							>
+								<a href="index.html" class=""> CC Campaign </a>
+							</li>
+						</ul>
+					</div>
+				</div>
+			</nav>
+
+			<!-- Mobile menu -->
+			<div
+				id="mobile-menu"
+				class="mobile-menu absolute top-0 hidden w-full origin-top animate-times-menu-animation flex-col justify-center bg-metal px-4 py-3 text-4xl text-white"
+			>
+				<button class="close self-start">X</button>
+				<div
+					class="flex h-screen-94h flex-col items-center gap-4 py-8"
+					aria-label="mobile-menu"
+				>
+					<a
+						href="index.html"
+						class="w-full nav-link px-4 text-center hover:opacity-70"
+						>Home</a
+					>
+
+					<a
+						href="./about.html"
+						class="w-full nav-link px-4 text-center hover:opacity-70"
+						>About</a
+					>
+					<a
+						href="#program"
+						class="nav-link w-full px-4 text-center hover:opacity-70"
+						>Program</a
+					>
+					<a
+						href="#"
+						class="nav-link w-full px-4 text-center hover:opacity-70"
+						>Join</a
+					>
+					<a
+						href="#"
+						class="nav-link w-full px-4 text-center hover:opacity-70"
+						>Sponsor</a
+					>
+					<a
+						href="#"
+						class="nav-link w-full px-4 text-center hover:opacity-70"
+						>News</a
+					>
+				</div>
+			</div>
```

Comment:
- [ ] Kindly wrap the navigation bar for the mobile version into the `nav` tag in order to structure your HTML semantically correctly :wink: